src/lib/server/endpoints/google/endpointVertex.ts

```diff
@@ -16,6 +16,7 @@ export const endpointVertexParametersSchema = z.object({
 	model: z.any(), // allow optional and validate against emptiness
 	type: z.literal("vertex"),
 	location: z.string().default("europe-west1"),
+	extraBody: z.record(z.any()).optional(),
```

Comment:
nit but if we're able to strictly type it, lets do it
```suggestion
	extraBody: z.object({ model_version: z.string() }).optional(),
```